locale/it_IT/locale.xml

```diff
@@ -1096,4 +1078,17 @@ Il processo di validazione pu&#x00F2; non funzionare sugli URL di ricerca che ri
 
 	<!-- Debug messages -->
 	<message key="debug.notes.currencyListLoad">Elenco delle valute caricate "{$filename}" da XML</message>
+	<message key="journal.viewAllIssues">Visualizza tutti i numeri</message>
+	<message key="user.noRoles.selectUsersWithoutRoles">Includi utenti che non hanno un ruolo in questa rivista</message>
+	<message key="user.role.journalAssistant">Redattore</message>
+	<message key="user.role.journalAssistants">Redattori</message>
+	<message key="user.role.guestEditor">Guest editor</message>
```

Comment:
Same here… manually entered that text or copied it from English file, but I am ok with keeping it.